src/world/area_osr/osr_03/AB62B0.c

```diff
@@ -24,6 +24,39 @@ ApiStatus func_802400C4_AB6354(Evt* script, s32 isInitialCall) {
 }
 
 
-INCLUDE_ASM(s32, "world/area_osr/osr_03/AB62B0", func_802401A0_AB6430);
+ApiStatus func_802401A0_AB6430(Evt* script, s32 isInitialCall) {
+    if (isInitialCall) {
+        script->functionTemp[1] = 0;
+        script->functionTemp[2] = 0;
+    }
+    switch (script->functionTemp[2]) {
+    case 0:
+        set_screen_overlay_color(0, 0xD0U, 0xD0U, 0xD0U);
```

Comment:
Colors and other values below in decimal please, plus remove the U after numbers